rotkehlchen/accounting/structures.py

```diff
@@ -406,3 +408,38 @@ def identifier(self) -> str:
             str(event_subtype)
         )
         return sha3(hashable.encode()).hex()
+
+
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class StakingEvent:
+    event_type: Literal[
+        'staking_deposit_asset',
+        'staking_receive_asset',
+        'staking_remove_asset',
+        'receive',
+    ]
+    asset: Asset
+    balance: Balance
+    timestamp: Timestamp
+    location: Location
+
+    @classmethod
+    def from_history_base_entry(cls, event: HistoryBaseEntry) -> 'StakingEvent':
+        event_type = str(event.event_subtype) if event.event_subtype is not None else 'receive'
```

Comment:
This here is because we forgot to add a subtype to the staking events and rewards are not categorized. We can assume that staking type without subtype is receive something or otherwise include an update in the data migration